src/main/java/nextstep/subway/path/application/GraphService.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.path.domain.SubwayGraph;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GraphService {
+
+    private SubwayGraph subwayGraph;
+
+    public GraphService(SubwayGraph subwayGraph) {
+        this.subwayGraph = subwayGraph;
+    }
+
+    public SubwayGraph findGraph() {
+        return subwayGraph;
```

Comment:
```suggestion
        return new SubwayGraph(findSections());
```

이 부분에서  SubwayGraph를 초기화 해주면 어떨까요?

Reply:
넵!